NDTensors/test/Project.toml

```
@@ -25,3 +25,6 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 [weakdeps]
 AMDGPU = "21141c5a-9bdb-4563-92ae-f87d6854732e"
 Metal = "dde4c033-4e86-420c-a63e-0dd931031962"
+
+[compat]
+Metal = "^1.1.0"
```

```suggestion
Metal = "1.1.0"
```
`^` is implied if left off, see https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers.

---

Thanks I will fix this and I am verifying that moving Metal to `[extras]` works properly. (I will also move AMDGPU to `[extras]` if it does)